### PR TITLE
Updated guides with new resteasy params

### PR DESCRIPTION
**Pull request by FroMage**
This is the guides part of https://github.com/quarkusio/quarkus-quickstarts/pull/329

### CODE REVIEW

File: docs/src/main/asciidoc/hibernate-search-guide.adoc [hunk 1]
```diff
@@ -475,8 +475,8 @@ In our existing `LibraryResource`, we just need to inject the following methods
     @GET
     @Path("author/search")
     @Transactional
-    public List<Author> searchAuthors(@QueryParam("pattern") String pattern, // <3>
-            @QueryParam("size") Optional<Integer> size) {
+    public List<Author> searchAuthors(@QueryParam String pattern, // <3>
```
Comment (gsmet) [inline docs/src/main/asciidoc/hibernate-search-guide.adoc]: Could you add in the callout that the `@QueryParam` here is the one from `org.jboss.resteasy.annotations.jaxrs.`?

**Review by gsmet — CHANGES_REQUESTED** [review]
Looks good to me. Added a small suggestion to make it extra clear which `@QueryParam` should be imported.

File: docs/src/main/asciidoc/hibernate-search-guide.adoc (hunk 1 above)
Comment (FroMage) [reply docs/src/main/asciidoc/hibernate-search-guide.adoc]: Done.

**Comment by FroMage** [comment]
Updated and rebased.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Merged, thanks!

**Pull request merged**